.travis.yml

```diff
@@ -14,55 +14,55 @@ matrix:
     env:
     - COMPILER=g++
     - IMG=gcc49
-    - CMAKE_EXTRA_FLAGS="-DENABLE_WARNINGS=On -DENABLE_TBB=On -DRAJA_DEPRECATED_TESTS=On"
+    - CMAKE_EXTRA_FLAGS="-DRAJA_ENABLE_WARNINGS=On -DRAJA_ENABLE_TBB=On -DRAJA_DEPRECATED_TESTS=On"
   - compiler: gcc6
     env:
     - COMPILER=g++
     - IMG=gcc6
-    - CMAKE_EXTRA_FLAGS="-DENABLE_WARNINGS=On -DENABLE_TBB=On -DRAJA_ENABLE_RUNTIME_PLUGINS=On"
+    - CMAKE_EXTRA_FLAGS="-DRAJA_ENABLE_WARNINGS=On -DRAJA_ENABLE_TBB=On -DRAJA_ENABLE_RUNTIME_PLUGINS=On"
   - compiler: gcc7
     env:
     - COMPILER=g++
     - IMG=gcc7
-    - CMAKE_EXTRA_FLAGS="-DENABLE_WARNINGS=On -DENABLE_TBB=On"
+    - CMAKE_EXTRA_FLAGS="-DRAJA_ENABLE_WARNINGS=On -DRAJA_ENABLE_TBB=On"
   - compiler: gcc8
     env:
     - COMPILER=g++
     - IMG=gcc8
-    - CMAKE_EXTRA_FLAGS="-DENABLE_WARNINGS=On -DENABLE_TBB=On -DRAJA_ENABLE_BOUNDS_CHECK=ON"
+    - CMAKE_EXTRA_FLAGS="-DRAJA_ENABLE_WARNINGS=On -DRAJA_ENABLE_TBB=On -DRAJA_ENABLE_BOUNDS_CHECK=ON"
   - compiler: clang9
     env:
     - COMPILER=clang++-9
     - IMG=clang9
-    - CMAKE_EXTRA_FLAGS="-DCMAKE_CXX_FLAGS=-fmodules -DENABLE_TBB=On"
+    - CMAKE_EXTRA_FLAGS="-DCMAKE_CXX_FLAGS=-fmodules -DRAJA_ENABLE_TBB=On"
```

Comment:
Should the "-fmodules" flag be here?

Reply:
I think we can keep this if it's working.